openstf/pipeline/predict_sklearn.py

```diff
@@ -49,14 +52,23 @@ def predict_pipeline(pj, input_data):
         horizons=[0.25], features=model._Booster.feature_names
     ).add_features(validated_data)
 
+    # Prep forecast input
+    forecast_input_data = data_with_features[forecast_start:forecast_end]
+
     # Check if sufficient data is left after cleaning
     if not validation.is_data_sufficient(data_with_features):
-        print("Use fallback model")
-
-    # Predict
-    forecast_input_data = data_with_features[forecast_start:forecast_end]
+        fallback_strategy = "extreme_day"  # this can later be expanded
+        logger.warning(
+            "Using fallback forecast",
+            forecast_type="fallback",
+            pid=pj["id"],
+            fallback_strategy=fallback_strategy,
+        )
+        model_forecast = generate_fallback(forecast_input_data, input_data.iloc[:, [0]])
```

Comment:
Instead of using inputdata.iloc[:, [0]] can we use inputdata['load']? This is maybe more readable. Moreover I think the 'load' column should always be available in this case. What do you think?

Reply:
I went back and forth on this, but I agree with you. I'll change it and include this requirement in the docstring